docs/reference/installation.rst

```diff
@@ -1,24 +1,43 @@
+.. index::
+    single: Installation
+    single: Configuration
+
 Installation
 ============
 
-SonataArticleBundle can be installed at any moment during a project's lifecycle,
-whether it's a clean Symfony installation or an existing project.
+Prerequisites
+-------------
 
-Downloading the Bundle
-----------------------
+PHP ^7.2 and Symfony ^4.4 are needed to make this bundle work, there are
+also some Sonata dependencies that need to be installed and configured beforehand.
 
-.. code-block:: bash
+Required dependencies:
 
-    composer require sonata-project/article-bundle
+* `SonataAdminBundle <https://sonata-project.org/bundles/admin>`_
+* `SonataClassificationBundle <https://sonata-project.org/bundles/classification>`_
+* `SonataMediaBundle <https://sonata-project.org/bundles/media>`_
+
+And the persistence bundle:
 
-Check `packagist <https://packagist.org/packages/sonata-project/article-bundle>`_
-for existing versions.
+* `SonataDoctrineOrmAdminBundle <https://sonata-project.org/bundles/doctrine-orm-admin>`_
```

Comment:
IMO, this section must be updated in order to show the remaining persistence bundles.

Reply:
I left only the implemented ones for the ArticleBundle